src/routes/jira/workspaces/jira-workspaces-get.ts

```diff
@@ -1,3 +1,62 @@
-export const JiraWorkspacesGet = () => {
-	// Content included in ARC-2115-search-connected-orgs
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { RepoSyncState } from "models/reposyncstate";
+import { Errors } from "config/errors";
+import { paginatedResponse } from "utils/paginate-response";
+
+export type Workspace = {
+	id: string,
+	name: string,
+	canCreateContainer: boolean
+}
+
+const DEFAULT_PAGE_NUMBER = 1; // Current page
+export const DEFAULT_LIMIT = 20; // Number of items per page
+
+const findMatchingOrgs = async (subscriptions: Subscription[], orgName?: string): Promise<Workspace[]>  => {
+	let matchingRepos;
+
+	if (orgName) {
+		matchingRepos = await Promise.all(subscriptions.map(async (subscription: Subscription) => {
+			return await RepoSyncState.findByOrgNameAndSubscriptionId(subscription, orgName);
+		}));
+	} else {
+		matchingRepos = await Promise.all(subscriptions.map(async (subscription: Subscription) => {
+			return await RepoSyncState.findOneFromSubscription(subscription);
+		}));
+	}
+
+	const matchedOrgs = matchingRepos
+		.filter(org => org !== null)
```

Comment:
\nit I suggest to use the [Boolean trick](https://michaeluloth.com/filter-boolean) for such null checks, its easier to read and use.
```suggestion
		.filter(Boolean)
```

Reply:
If we use that we get TS errors when trying to deconstruct `const { subscriptionId, repoOwner } = org;` So we either go with something like this or use a non-null assertion operator `const { subscriptionId, repoOwner } = org!; // Use non-null assertion operator`. I'd prefer to filter out null values as non-null assertions have the potential to lead to runtime errors.